BlockEntity/BEFruitpress.cs

```diff
@@ -497,7 +489,7 @@ public bool OnBlockInteractStep(float secondsUsed, IPlayer byPlayer, EnumFruitPr
                 RunningAnimation anim = animUtil.animator.GetAnimationState("compress");
                 if (anim != null)
                 {
-                    udpateSqueezeRel(anim);
+                    updateSqueezeRel(anim);

```

Comment:
Use this since updateSqueezeRel checks for null already
```cs
updateSqueezeRel(animUtil.animator.GetAnimationState("compress"));
```

Reply:
Oh, you're right, thanks!